### PR TITLE
[CORDA-2219] Show message if CorDapp already exists

### PR DESCRIPTION
**Pull request by gatsinski**
This pull request attempts to fix [CORDA-2219](https://r3-cev.atlassian.net/browse/CORDA-2219)

### CODE REVIEW

**Comment by gatsinski** [comment]
Looks like the tests are failing. I'll examine the problem.

**Comment by vkolomeyko** [comment]
Looks fine to me, looping in @shamsasari for second opinion.
This API stability TC job failure looks rather odd. Let me re-run it.

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/network/NetworkBootstrapper.kt [hunk 1]
```diff
@@ -226,7 +227,13 @@ internal constructor(private val initSerEnv: Boolean,
             println("Copying CorDapp JARs into node directories")
             for (nodeDir in nodeDirs) {
                 val cordappsDir = (nodeDir / "cordapps").createDirectories()
-                cordappJars.forEach { it.copyToDirectory(cordappsDir) }
+                cordappJars.forEach {
+                    try {
+                        it.copyToDirectory(cordappsDir)
+                    } catch (e: FileAlreadyExistsException) {
+                        println("WARNING: ${it.fileName} already exists in $cordappsDir")
```
Comment (shamsasari) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/network/NetworkBootstrapper.kt]: Add "..., ignoring and leaving existing CorDapp untouched"

Comment (gatsinski) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/network/NetworkBootstrapper.kt]: Done.

**Comment by shamsasari** [comment]
Why are we changing the api file? The change is trivial and does not need such a change. I can't approve this modification. What was the original issue?

**Comment by vkolomeyko** [comment]
Yes, API change is a bit unexpected.

Looks like `diff` tool got very confused and was claiming that `equals()` method for one of the `core` classes been removed and added back.

I discussed with @chrisr3 offline and we could not think of any better solution than updating `api-current.txt` for the class in question with current state in `master` branch. Earlier on people been adding methods to this class but did not update `api-current.txt`.

**Pull request merged**